packages/flutter_test/lib/src/controller.dart

```diff
@@ -234,6 +235,32 @@ abstract class WidgetController {
   /// A reference to the current instance of the binding.
   final WidgetsBinding binding;
 
+  /// The [TestPlatformDispatcher] that is being used in this test.
+  ///
+  /// This will be injected into the framework such that calls to
+  /// [WidgetsBinding.platformDispatcher] will use this. This allows
+  /// users to change platform specific properties for testing.
+  ///
+  /// See also:
+  ///
+  ///   * [TestFlutterView] which allows changing view specific properties
+  ///     for testing
+  ///   * [implicitView] and [viewOf] which are used to find
+  ///     [TestFlutterView]s from the widget tree
+  TestPlatformDispatcher get platformDispatcher => binding.platformDispatcher as TestPlatformDispatcher;
+
+  /// The implicit [TestFlutterView] provided when the platform does not support
```

Comment:
Where will that view come from in that case?

Right now, this `implicitView` wraps the actual `implicitView` from `TestPlatformDispatcher`, which in turn is wrapping a real `PlatformDispatcher`. I'm not seeing how it _can_ be different from what is supported by `PlatformDispatcher`.

Do you have a specific suggestion for the docs? I'm not sure my understanding of the plans is up to snuff to resolve your concern here.

Reply:
I took a pass at updating the doc comment here. Let me know if I got the right idea across.

Reply:
See my comment below. I think this shouldn't mention window support at all and instead just say that this is where `pumpWidget` will draw the provided widget into. Should also mention that it can be used to mock certain global properties.

If you'd like I can write up a concrete suggestion. Let me know!

Reply:
Think I got it this time. Resolving this convo, but feel free to continue this in the other thread if I missed something!